src/web/theme/modules/Header/Navigation/Navigation.scss

```diff
@@ -0,0 +1,28 @@
+@keyframes fadeIn {
+  from {
+    display: unset;
+    opacity: 0;
+  }
+
+  to {
+    display: unset;
+    opacity: 1;
+  }
+}
+
+.navigation {
+  &__linklist {
+    box-shadow: 1px 1px 10px $shade03;
+    background: white;
+    animation-duration: 0.5s;
+    animation-name: fadeIn;
+    opacity: 1;
+    position: absolute;
+    right: 1em;
+    top: 1em;
+    display: unset;
```

Comment:
Why do you need this here ? it also looks duplicated in the animation above. Am I missing something ?